### ISSUE TITLE
User account area

### ISSUE THREAD

**Issue by razvanaldea89** [WPChill/download-monitor#1228]
An user account area where an user can see what, how many times, when it downloaded files -> the Reports functionality could be used here.

Also, the allowed Downloads and non-allowed Downloads can be shown here.

@giucu91 @TeoAlex @cristianraiber thoughts?

**Comment by razvanaldea89** [comment]
Also, this can get in touch with other extensions, like AAM, and show the user how many downloads he has remaining ( if it's a global restriction rule for example )

**Comment by cristianraiber** [comment]
I don't think this should be a separate extension. I think this should be core functionality.

**Comment by jagge72** [comment]
Does this mean you can now display downloads left per user role?

**Comment by razvanaldea89** [comment]
The functionality is not implemented yet, but in the future each user will have an account where more information will be shown regarding download status and such.

Warmly,
Razvan
